apps/condo/package.json

```diff
@@ -23,7 +23,8 @@
     "lint-schema": "NODE_ENV=test node ./../../bin/lint-schema.js",
     "lint-i18n-translations": "node ./../../bin/lint-i18n-translations.js",
     "refresh-sbbol-tokens": "node ./../../bin/sbbol.js",
-    "create-integration": "node ./../../bin/create-integration.js"
+    "create-integration": "node ./../../bin/create-integration.js",
+    "sbbol:crypto:get": "node ./../../bin/sbbol/crypto.js"
```

Comment:
maybe, 
```
sbbol-crypto-get
```

Reply:
I have proposed to standardise script names.
https://doma.atlassian.net/browse/DOMA-1608